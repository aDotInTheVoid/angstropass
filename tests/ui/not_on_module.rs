#[nanopass::langs]
pub fn lol() {}

fn main() {}
