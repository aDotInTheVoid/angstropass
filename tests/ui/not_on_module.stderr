error: expected `mod`
 --> tests/ui/not_on_module.rs:2:5
  |
2 | pub fn lol() {}
  |     ^^
